src/App.jsx

```diff
@@ -1,12 +1,31 @@
-import { useState } from 'react'
+import Nav from './components/Nav'
+import { Route, Routes } from 'react-router-dom'
+import { useEffect, useState } from "react";
+import { getTopics } from "../api";
+import HomeContainer from "./components/HomeContainer"
+//import ArticleContainer from './components/ArticleContainer';
 
 import './App.css'
 
 function App() {
 
+  const [topics, setTopics] = useState([])
+
+  useEffect(() => {
```

Comment:
I'd recommend doing this call in the Nav as that's the only component using the data right now.